demo/api/src/config/config.ts

```diff
@@ -1,7 +1,7 @@
+import cometConfig from "@src/../comet-config.json";
```

Comment:
comet-config.json is located outside src, this is not ideal but maybe better than `../../`

Reply:
Why not move it into the source folder, as it's part of the source code?

Reply:
I remember it was in src at some stage and explicitly moved outside. We can move it in again :)

(out of scope for this PR)